LEAF_Request_Portal/js/vue_conditions_editor/LEAF_conditions_editor.js

```diff
@@ -584,181 +312,349 @@ const ConditionsEditor = Vue.createApp({
+        return f.split('\n')[0].trim();
+    },
+    /**
+    * @returns list of indicators that are on the same page, enabled as parents, and different than child 
+    */
+    selectableParents() {
+      let selectable = [];
+      const headerIndicatorID = this.childIndicator?.headerIndicatorID || 0;
+      if (headerIndicatorID !== 0) {
+        selectable = this.indicators.filter(i => {
+          const parFormat = i.format?.split('\n')[0].trim().toLowerCase();
+          return i.headerIndicatorID === headerIndicatorID &&
+              parseInt(i.indicatorID) !== this.childIndID &&
+              this.enabledParentFormats.includes(parFormat);
+        });
+      }
+      return selectable;
+    },
+    /**
+    * @returns list of operators and human readable text base on parent format
+    */
+    selectedParentOperators() {
+      let operators = [];
+      switch(this.parentFormat) {
+        case 'multiselect':
+        case 'checkboxes':
+          operators = [
+              {val:"==", text: "includes"},
+              {val:"!=", text: "does not include"}
+          ];
+          break;
+        case 'dropdown':
+        case 'radio':
+        default:
+          operators = [
+              {val:"==", text: "is"},
+              {val:"!=", text: "is not"}
+          ];
+          break;
+      }
+      return operators;
+    },
+    crosswalkLevelTwo() {
+      let levelOptions = [];
+      const headerIndicatorID = this.childIndicator?.headerIndicatorID || 0;
+      if(headerIndicatorID !== 0) {
+        levelOptions = this.indicators.filter((i) => {
+          const format = i.format?.split("\n")[0].trim().toLowerCase();
+          return (
+              i.headerIndicatorID === headerIndicatorID &&
+              parseInt(i.indicatorID) !== this.childIndID &&
+              ['dropdown', 'multiselect'].includes(format)
+          );
+        });
+      }
+      return levelOptions;
+    },
+    /**
+    * @returns list of options for comparison based on parent indicator selection
+    */
+    selectedParentValueOptions() {
+      const fullFormatToArray = (this.selectedParentIndicator?.format || '').split("\n");
+      let options = fullFormatToArray.length > 1 ? fullFormatToArray.slice(1) : [];
+      options = options.map(o => o.trim());
+      return options.filter(o => o !== '');
+    },
+    /**
+    * @returns list of options for prefill outcomes.  Does NOT combine with file loaded options.
+    */
+    selectedChildValueOptions() {
+      const fullFormatToArray = (this.childIndicator?.format || '').split("\n");
+      let options = fullFormatToArray.length > 1 ? fullFormatToArray.slice(1) : [];
+      options = options.map(o => o.trim());
+      return options.filter(o => o !== '');
     },
     canAddCrosswalk() {
       return (this.childFormat === 'dropdown' || this.childFormat === 'multiselect')
     },
+    childPrefillDisplay() {
+      let returnVal = '';
+      switch(this.childFormat) {

```

Comment:
Likely want toLower() here.

Reply:
childFormat and parentFormat are computed to return lower case (conditions.selectedOutcome too).